build.gradle

```diff
@@ -112,6 +112,8 @@ dependencies {
   testImplementation "org.scalatestplus:mockito-3-12_$scalaVersion:3.2.10.0" // syntactic sugar
 }
 
+test.dependsOn processScoverageResources
```

Comment:
I'm really wondering why this is necessary here, but not in SIMONA. Have not found an answer so far

Reply:
Ok, reason could be that we configure tests in simona in `tests.gradle`. Suffices as an explanation to me